replicatedhq/ship
# 15590 app settings typing

What I Did
------------
Fix glitchy typing on application settings screen

How I Did it
------------
Save value in state and only update the state from the props if the input is not focused.

How to verify it
------------
Using the hosted version on replicated.com/ship ship init this app
`replicated.app/travis?customer_id=8627b82efe834d9256e33a5c58b72164&installation_id=EPKWEO2C-xuspKL1YHGole5SJeNXgyg4`

Click through until you get to the application settings screen and then type in any of the inputs. You should be able to type and backspace freely without the cursor jumping around or loosing some of the text you type.

Description for the Changelog
------------
Fix buggy typing on application settings page


Picture of a Boat (not required but encouraged)
------------
